e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts

```diff
@@ -116,6 +118,9 @@ class RegistrationDetails {
     this.personAffectedHouseNumber = this.page.getByTestId(
       'update-property-item-numeric-input',
     );
+    this.personAffectedInputForm = this.page.getByTestId(
+      'update-property-item-input-form',
+    );
   }
 
```

Comment:
The Lines 112-114 above can also be removed, as there are no more users of the `this.personAffectedCustomAttribute`

Reply:
This one is actually used. I think you meant the other locator and corresponding lines. Those were removed.